src/ts/content/extensions/util/onLogged.ts

```diff
@@ -17,9 +18,11 @@ interface OnLogOptions {
 	description?: string;
 }
 
+type LoggedStatus = {authed: boolean; sheetSet: boolean; ready: boolean};
```

Comment:
ooo cool

i recall being told once that keeping track of states like this could be done with an enum so that every state is mutually exclusive, but i don't think that's necessary here AND `ready` is only true if the other 2 things are true

ALSO this is probably useful for debugging

nvm ignore me

Reply:
um this probably isn't the right way to do this.... but it was ez